en/micro-integrator/docs/references/connectors/mongodb-connector/mongodb-connector-example.md

```diff
@@ -0,0 +1,257 @@
+
+    <img src="../../../../assets/img/connectors/mongodb-conn-2.png" title="Find function" width="800" alt="Find function"/>
+
+If you do not want to configure this yourself, you can simply [get the project](#get-the-project) and run it.
+
+## Connect to MongoDB Atlas
+
+1. In the Clusters view, click Connect for the cluster to which you want to connect.
+
+2. Click Choose a connection method.
+
+3. Click Connect your application.
+
+4. Select Java from the Driver dropdown.
+
+5. Select your version of the driver from the dropdown.
+
+6. Uncheck "Include full driver code example" checkbox to get the connection string.
+
+## Configure the connector in WSO2 Integration Studio
+
+Follow these steps to set up the Integration Project and the Connector Exporter Project.
+
+{!references/connectors/importing-connector-to-integration-studio.md!}
+
+## Creating the Integration Logic
+
+1.  Create a new integration project named `MongodbConnector`. Be sure to enable a Connector Exporter.
+
+    <img src="../../../../assets/img/connectors/mongodb-conn-3.png" title="Create project" width="500" alt="Create project"/>
+
+2.  Right click on the created Integration Project and select, -> **New** -> **Rest API** to create the REST API.
+
+    <img src="../../../../assets/img/connectors/adding-an-api.png" title="Adding a Rest API" width="800" alt="Adding a Rest API"/>
+
+3.  Provide the API name as `MongoConnector` and the API context as `/mongodbconnector`.
+
+4.  First we will create the `/insertmany` resource. This API resource will insert documents into the MongoDB database.<br/>
+    Right click on the API Resource and go to **Properties** view. We use a URL template called `/insertmany` as we have two API resources inside single API. The method will be `Post`.
+
+    <img src="../../../../assets/img/connectors/mongodb-conn-4.png" title="Adding the API resource." width="800" alt="Adding the API resource."/>
+
+5.  Drag and drop the 'insertMany' operation of the MongoDB Connector to the Design View as shown below.
+
+    <img src="../../../../assets/img/connectors/mongodb-conn-5.png" title="Adding the insert many operation." width="800" alt="Adding the insert many operation."/>
+
+6.  Create a connection from the properties window by clicking on the '+' icon as shown below.
+
+    Following values can be provided to connect to MongoDB database. <br/>
+
+    - Connection Name - connectionURI
+    - Connection Type - URI
+    - Connection URI - mongodb+srv://server.example.com/?connectTimeoutMS=300000&amp;authSource=aDifferentAuthDB
+    - Database - TestDatabase
+
+    <img src="../../../../assets/img/connectors/mongodb-conn-6.png" title="Adding the connection." width="800" alt="Adding the connection."/>
+
+7.  After the connection is successfully created, select the created connection as 'Connection' from the drop down in the properties window.
```

Comment:
"After the connection is successfully created, you can select the new connection from the <field_name> menu in the properties view."

Please replace <field_name> with the the correct name. 
Also, I am assuming this is the properties 'view' and not a dialog box or window?

Reply:
Yes, this is a properties view.